modules/kernel/src/main/scala-3/tofu/internal/instances/ClockInstance.scala

```diff
@@ -1,14 +1,26 @@
 package tofu.internal
 package instances
 
+import cats.~>
+import tofu.higherKind.{RepK, RepresentableK}
 import tofu.internal.carriers.ClockCE3Carrier
 import tofu.internal.carriers.ClockCE2Carrier
 import tofu.time.Clock
 
+import java.util.concurrent.TimeUnit
 import scala.compiletime.summonFrom
 
-private[tofu] trait ClockInstance:
```

Comment:
all `*Instance` files uses optinal braces syntax. Could you return it back?

Reply:
i've removed the braces